certified-connectors/SigningHub/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1936 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "SigningHub is a complete solution for document approval workflows, advanced digital signatures. It is designed to quickly optimize the way businesses send, review, approve and sign their business documents.",
+    "title": "SigningHub",
+    "version": "1.2.0",
+		"contact":{
+			"url": "https://www.signinghub.com/contact-us/",
+			"name": "Support",
+			"email": "support@ascertia.com"
+	}
+  },
+  "host": "office.signinghub.com",
+  "basePath": "/Flow/v12",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/v1/about": {
+      "get": {
+        "description": "Get the SigningHub instance details",
+        "operationId": "AboutSigningHub",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "default response",
+            "schema": {
+              "properties": {
+                "aboutSigningHub": {
+                  "description": "About SigningHub",
+                  "properties": {
+                    "build": {
+                      "description": "Build of SigningHub instance",
+                      "title": "Build",
+                      "type": "string",
+                      "x-ms-visibility": "important"
+                    },
+                    "copyright": {
+                      "description": "Copyright of SigningHub",
+                      "title": "Copyright",
+                      "type": "string",
+                      "x-ms-visibility": "important"
+                    },
+                    "installation_name": {
+                      "description": "Installation name of SigningHub instance",
+                      "title": "Installation name",
+                      "type": "string",
+                      "x-ms-visibility": "important"
+                    },
+                    "patents": {
+                      "description": "This containc SigningHub US patent number",
```

Comment:
All descriptions must be free from grammatical errors. Please review all descriptions, summaries, etc.

Reply:
removed spell and grammatical errors